clients/cloud/python/ccloud_lib.py

```diff
@@ -53,18 +52,24 @@ def __init__(self, name=None):
         # Do *not* include in the serialized object.
         self.id = uuid4()
 
+    @staticmethod
+    def dict_to_name(obj, ctx):
+        return Name(obj['name'])
+
+    @staticmethod
+    def name_to_dict(name, ctx):
+        return Name.to_dict(name)
+
     def to_dict(self):
         """
             The Avro Python library does not support code generation.
             For this reason we must provide a dict representation of our class for serialization.
         """
-        return {
-            "name": self.name
-        }
+        return dict(name=self.name)
```

Comment:
is there a reason to change the literal to a dict constructor? I'd use a literal and put it on one line `return { "name": self.name }`. but kinda bike shedding.

Reply:
I modeled these changes after the official examples: https://github.com/confluentinc/confluent-kafka-python/blob/master/examples/avro_consumer.py#L62